test/accelerators/NNPA/numerical/TestGRU-NNPA.cpp

```diff
@@ -0,0 +1,100 @@
+/*
```

Comment:
Could you explain in a top comment how different the TestGRU-NNPA.cpp file is from the regular TestGRU.cpp? It was not obviously clear to me what the differences are?

please do so for this and the other TestXXX-NNPA files in this dir

Reply:
I added comments in TestLSTM-NNPA.cpp and TestGRU-NNPA.cpp. Following sentence is for TestGRU-NNPA.cpp.

```
// This file contains configurations for testing GRU models on NNPA.
// This file is based on test/numerical/TestGRU.cpp, but includes NNPA-specific
// configurations. For example, bidirectional GRU is not included because it is
// not supported in onnx-mlir. LinierBeforeReset is disabled because the zDNN
// library used in NNPA does not support it.
```